### PR TITLE
Chore vpcep

### PR DESCRIPTION
**Pull request by hmbig2**
**What this PR does / why we need it**:
1. move to services 
2. chore vpcep
**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note

```

## PR Checklist

* [X] Tests added/passed.
* [X] Documentation updated.
* [ ] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST='./huaweicloud/services/acceptance/vpcep' TESTARGS='-run=TestAcc'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/vpcep -v -run=TestAcc -timeout 360m -parallel 4
=== RUN   TestAccVPCEndpointApproval_Basic
=== PAUSE TestAccVPCEndpointApproval_Basic
=== RUN   TestAccVPCEndpoint_Basic
=== PAUSE TestAccVPCEndpoint_Basic
=== RUN   TestAccVPCEndpoint_Public
=== PAUSE TestAccVPCEndpoint_Public
=== RUN   TestAccVPCEPService_Basic
=== PAUSE TestAccVPCEPService_Basic
=== RUN   TestAccVPCEPService_Permission
=== PAUSE TestAccVPCEPService_Permission
=== CONT  TestAccVPCEndpointApproval_Basic
=== CONT  TestAccVPCEPService_Basic
=== CONT  TestAccVPCEndpoint_Public
=== CONT  TestAccVPCEndpoint_Basic
--- PASS: TestAccVPCEndpoint_Public (26.59s)
=== CONT  TestAccVPCEPService_Permission
--- PASS: TestAccVPCEPService_Basic (192.11s)
--- PASS: TestAccVPCEndpoint_Basic (202.14s)
--- PASS: TestAccVPCEndpointApproval_Basic (209.90s)
--- PASS: TestAccVPCEPService_Permission (197.73s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/vpcep     224.386s
```

### CODE REVIEW

File: huaweicloud/services/acceptance/vpcep/resource_huaweicloud_vpcep_endpoint_test.go [hunk 1]
```diff
@@ -79,57 +87,13 @@ func TestAccVPCEndpoint_Public(t *testing.T) {
...
-	for _, rs := range s.RootModule().Resources {
-		if rs.Type != "huaweicloud_vpcep_endpoint" {
-			continue
-		}
-
-		_, err := endpoints.Get(vpcepClient, rs.Primary.ID).Extract()
-		if err == nil {
-			return fmtp.Errorf("VPC endpoint still exists")
-		}
+		return nil, fmt.Errorf("error creating IoTDA v5 client: %s", err)
```
Comment (ShiChangkuo) [inline huaweicloud/services/acceptance/vpcep/resource_huaweicloud_vpcep_endpoint_test.go]: the error message is not correct

File: huaweicloud/services/acceptance/vpcep/resource_huaweicloud_vpcep_service_test.go [hunk 2]
```diff
@@ -92,57 +101,13 @@ func TestAccVPCEPService_Permission(t *testing.T) {
...
-	for _, rs := range s.RootModule().Resources {
-		if rs.Type != "huaweicloud_vpcep_service" {
-			continue
-		}
-
-		_, err := services.Get(vpcepClient, rs.Primary.ID).Extract()
-		if err == nil {
-			return fmtp.Errorf("VPC endpoint service still exists")
-		}
+		return nil, fmt.Errorf("error creating IoTDA v5 client: %s", err)
```
Comment (ShiChangkuo) [inline huaweicloud/services/acceptance/vpcep/resource_huaweicloud_vpcep_service_test.go]: ditto

File: huaweicloud/services/acceptance/vpcep/resource_huaweicloud_vpcep_approval_test.go [hunk 3]
```diff
@@ -1,41 +1,44 @@
...
+		ProviderFactories: acceptance.TestAccProviderFactories,
+		CheckDestroy:      rc.CheckResourceDestroy(),
 		Steps: []resource.TestStep{
 			{
 				Config: testAccVPCEndpointApproval_Basic(rName),
 				Check: resource.ComposeTestCheckFunc(
-					testAccCheckVPCEPServiceExists("huaweicloud_vpcep_service.test", &service),
-					testAccCheckVPCEndpointExists("huaweicloud_vpcep_endpoint.test", &endpoint),
-					resource.TestCheckResourceAttrPtr(resourceName, "id", &service.ID),
-					resource.TestCheckResourceAttrPtr(resourceName, "connections.0.endpoint_id", &endpoint.ID),
```
Comment (ShiChangkuo) [inline huaweicloud/services/acceptance/vpcep/resource_huaweicloud_vpcep_approval_test.go:31]: suggest to use `TestCheckResourceAttrPair` here

File: huaweicloud/services/acceptance/vpcep/resource_huaweicloud_vpcep_endpoint_test.go (hunk 1 above)
Comment (hmbig2) [reply huaweicloud/services/acceptance/vpcep/resource_huaweicloud_vpcep_endpoint_test.go]: fixed

File: huaweicloud/services/acceptance/vpcep/resource_huaweicloud_vpcep_approval_test.go (hunk 3 above)
Comment (hmbig2) [reply huaweicloud/services/acceptance/vpcep/resource_huaweicloud_vpcep_approval_test.go:31]: ok, done

File: huaweicloud/services/acceptance/vpcep/resource_huaweicloud_vpcep_service_test.go (hunk 2 above)
Comment (hmbig2) [reply huaweicloud/services/acceptance/vpcep/resource_huaweicloud_vpcep_service_test.go]: fixed

**Pull request merged**